app/assets/javascripts/device_stories.js

```diff
@@ -0,0 +1,10 @@
+window.addEventListener("load", function (e) {
+  ["1", "2", "3"].forEach(function (selection) {
```

Comment:
Seems like we could make this a little clearer than 1,2,3

Reply:
How about this? Changed to use `id` value directory.

Reply:
It's a little better.

I'm wondering why it's not more like https://www.bigbinary.com/blog/data-behavior#data-behavior-to-the-rescue though - seems like basically we're just binding a behavior to navigate to the currently selected param.

That might allow for adding/removing timeframes without potentially breaking the JS.

Reply:
Changed. How about now?